File: ICPC/WarmUp 10/ncpc2015solutions/conundrum_lukas.cs
//Solution by lukasP (Lukáš Poláček)
using System;
using System.Linq;

class Problem
{
    public static void Main ()
    {
        string s = Console.ReadLine();
        Console.WriteLine(Enumerable.Range(0, s.Length).Count(i => "PER"[i % 3] != s[i]));
    }
}

File: ICPC/WarmUp 10/ncpc2015solutions/conundrum_lukas.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# Solution by lukasP (Lukáš Poláček)

from sys import stdin
s = stdin.readline().strip()
print(sum([1 if "PER"[i % 3] != s[i] else 0 for i in range(len(s))]))
